inc/REST/RemoteDataController.php

```diff
@@ -77,6 +77,7 @@ public static function execute_query( WP_REST_Request $request ): array|WP_Error
 		return array_merge(
 			[
 				'block_name' => $block_name,
+				'data_source' => ConfigStore::get_data_source( $block_name ),
```

Comment:
same here

Reply:
Removed it. I was previously passing the `data_source` prop, but now I'm using the block name to retrieve the data source instead.